noir_stdlib/src/array.nr

```diff
@@ -83,3 +113,14 @@ impl<T, N> [T; N] {
         ret
     }
 }
+
+// helper function used to look up the position of a value in an array of Field
+unconstrained fn sort_helper<N>(a: [Field;N], find: Field) -> Field {
```

Comment:
Can you rename this to `find_index` or similar?

Reply:
Done